packages/website/src/content/docs/setup/sveltekit.mdx

```diff
@@ -9,16 +9,21 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 This guide gets Spotlight up and running in SvelteKit with Sentry. 
 You don't strictly need Sentry for Spotlight to work, we highly recommend it.
 
-## Install Sentry 
+## 1. Installation
+
+#### Install Sentry
```

Comment:
Is there a reason we're skipping the h3 level (`###`) here?

Reply:
No, there isn't a specific reason for choosing h4 @Lms24 
The visual difference in sizes between h3 and h2 was minimal, making it difficult to distinguish on the UI, i.e. why I opted for h4 to ensure clearer visual hierarchy.

Reply:
Ah I see, yeah no worries then let's leave it as is. Maybe we should make the difference between h2 and h3 stronger in general but let's not do this in this PR.